Source/Scene/Particle.js

```diff
@@ -26,13 +28,15 @@ define([
      * @param {Number} [options.mass=1.0] The mass of particles in kilograms.
      * @param {Cartesian3} [options.position=Cartesian3.ZERO] The initial position of the particle in world coordinates.
      * @param {Cartesian3} [options.velocity=Cartesian3.ZERO] The velocity vector of the particle in world coordinates.
-     * @param {Number} [options.life=Number.MAX_VALUE] The life of particles in seconds.
+     * @param {Number} [options.life=Number.MAX_VALUE] The life of the particle in seconds.
      * @param {Object} [options.image] The URI, HTMLImageElement, or HTMLCanvasElement to use for the billboard.
+     * @param {Object} [options.color] Sets the start and end colors.
      * @param {Color} [options.startColor=Color.WHITE] The color of a particle when it is born.
      * @param {Color} [options.endColor=Color.WHITE] The color of a particle when it dies.
+     * @param {Object} [options.scale] Sets the start and end scales.
```

Comment:
I think for `particle` itself, we should just stick with `start` and `end`?

Reply:
hmm i see your point but I also just realized there's no way of knowing a particular particle's current scale and color (the non interpolated ones) do you think it would be a good idea to actually add actual particle variables and getters for those for user debugging?

Reply:
I would say don't, since I think the only use case right now is for debugging.

Reply:
sounds good